### PR TITLE
[Storage] Support share Accesstier

### PR DESCRIPTION
**Pull request by blueww**
## Description

design pass review in pass review in: Azure/azure-powershell-cmdlet-review-pr#567
the code change already in storage preview release branch with PR https://github.com/Azure/azure-powershell/pull/12013

The feature target 10/27 release, anyway, it would be great if can release in 10/13 release.

## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [x] For public API changes to cmdlets:
    - [x] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [x] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/Storage/Storage.Management/ChangeLog.md [hunk 1]
```diff
@@ -18,6 +18,9 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
+* Support create/update file share with access tier
+    - `New-AzRmStorageShare`
+    - `Update-AzRmStorageShare`
```
Comment (isra-fel) [inline src/Storage/Storage.Management/ChangeLog.md:23]:
```suggestion
    - `New-AzStorageShare`
    - `Update-AzStorageShare`
```

**Review by isra-fel — CHANGES_REQUESTED** [review]
LGTM with a small typo

File: src/Storage/Storage.Management/ChangeLog.md (hunk 1 above)
Comment (blueww) [reply src/Storage/Storage.Management/ChangeLog.md:23]: The change log is right.
We have 2 set of File share cmdlet:
1. Dataplane , like New-AzStorageShare
2. Mgmt plane, like New-AzRmStorageShare.

The new properties only support on Mgmt plane, so should change the cmdlet name like "*-RmStorageShare"

**Review by isra-fel — APPROVED** [review]
I see. Thanks for the explaination!

**Pull request merged**